a10_octavia/controller/worker/tasks/service_group_tasks.py

```diff
@@ -41,11 +41,13 @@ def set(self, set_method, pool, vthunder, update=False):
                        lb_method=lb_method,
                        service_group_templates=service_group_temp,
                        axapi_args=args)
-            LOG.debug("Pool created successfully: %s", pool.id)
+            LOG.debug("Pool created/updated successfully: %s", pool.id)
             return pool
         except Exception as e:
```

Comment:
Instead of this. Let's change this to simply not catch the error here

Reply:
Yes. This is better. Have handled this change.